src/core/context/index.ts

```diff
@@ -72,7 +72,7 @@ export class Context implements AbstractContext {
     throw new ContextCancelation({ reason: 'Context Cancel' })
   }
 
-  log(level: LogLevel, message: string, extras?: object): void {
+  log(level: LogLevel, message: string, extras?: any): void {
```

Comment:
Do we require an object or can it actually be anything? If it needs to be an object maybe something like `Record<string, any>` would be more accurate and appease the linters.

Reply:
I agree that it's not ideal, but I assume that extras could also be a string? I see it's being used logged artifacts from error handling, which were very loosely-typed, so I didn't want to refactor any more code than I absolutely needed -- I'm kind of afraid to make any assumptions in the catch clauses.

Reply:
As long as we don't require it to be in the shape of an object and will allow anything, then `any` seems fine. Right now we would have had TypeScript enforce it as at least an object, but ok with loosening if that better reflects the actual state of things.

Reply:
Thanks for bringing this up. I changed this back, I think  it's safe to tighten this back up to at least object and then do unsafe coercions in the catch clause (until someone wants to refactor later).